bower.json

```diff
@@ -25,6 +25,6 @@
   "dependencies": {
     "medium-editor": "^5.0.0",
     "to-markdown": "^2.0.1",
-    "uglify-js": "^2.5.0"
+    "UglifyJS2": "https://github.com/mishoo/UglifyJS2.git#^2.5.0"
```

Comment:
Can we move this in `devDependencies` maybe (if there are no `devDependencies` field, create it)?

Also, is there a specific reason why you chose to use the git url (:bulb: **Tip**: You can directly do `"mishoo/UglifyJS2#^2.5.0"`)? :grin: